workspaces/src/rpc/client.rs

```diff
@@ -96,7 +96,7 @@ impl Client {
         };
 
         let result = std::str::from_utf8(&call_result)?;
-        Ok(serde_json::from_str(result)?)
+        Ok(result.into())
```

Comment:
This should probably be bytes. View calls can return anything, and esp when using borsh returns this will be unusable

Reply:
Changed this to use `ViewResultDetails` instead